### PR TITLE
Noinline attribute when using llvm-print-ir

### PR DESCRIPTION
**Pull request by coodie**
In full optimization stage function sometimes was inlined and removed from llvm module, easiest way to fix this is to add 'noinline' attribute for it.

The preferred way however is to add noinline pragma before function we want to print so that this option doesn't interfere with other optimization options and it is explicitly stated in code.

Other solution would be add _llvm-print-inline_ option which would specify if we want to turn this off or on.

Anyway, now we can see that adding noinline attribute is very easy to do.

### CODE REVIEW

File: compiler/codegen/symbol.cpp [hunk 1]
```diff
@@ -56,6 +54,10 @@
 #include <inttypes.h>
 #include <stdint.h>
 
+#ifdef HAVE_LLVM
+#include "llvm/IR/Attributes.h"
+#endif
```
Comment (coodie) [inline compiler/codegen/symbol.cpp]: I don't like putting these ifdefs here, but I can't see the default place where to put these ifdefs.

Comment (mppf) [inline compiler/codegen/symbol.cpp]: This is OK. I'd probably put the LLVM include before the first system include (`<algorithm>`). compiler/include/llvmUtil.h might work too but it's probably smarter to just put the include you need for this file here. Speaking of that, I'm surprised that the other llvm:: stuff in this file worked addFnAttr didn't. Are you sure you can't just remove the new include entirely?

Comment (coodie) [inline compiler/codegen/symbol.cpp]: Well, apparently it does, I can recall that it didn't compile without this header but apparently my crazy mind made it up.

**Comment by mppf** [comment]
Passed full local testing.

**Pull request merged**